.eslintignore

```diff
@@ -5,3 +5,4 @@ storybook-static
 coverage
 docs
 webpack.e2e.config.js
+src/e2e-test-utils.js
```

Comment:
why is this one ignored? if it contains some local settings, we may need to split it

Reply:
Yeah, the reason was because I did not want to write sauce labs capabilities specific flow type definitions, if we are going to remove this file with Nightwatch anyways. Do you think this is a blocker?

Reply:
Might want to consider ignoring inline in the file in that case, with either `/* eslint-disable */`, or a `/* no-flow */` comment.

Reply:
that makes perfect sense

Reply:
ok, seems like the whole is little early to ignore so far